nixos/tests/pleroma.nix

```diff
@@ -205,7 +205,7 @@ import ./make-test-python.nix ({ pkgs, ... }:
         };
         postgresql = {
           enable = true;
-          package = pkgs.postgresql_12;
+          package = pkgs.postgresql_13;
```

Comment:
That would be ideal but for future reference in case anyone is reading this later on, it does depend on the ecosystem. With something like postgresql, I guess it wouldn't affect much but with something like LLVM or Flutter, lots of major changes can make it difficult to upgrade so those should be set to a fixed version.

Reply:
I think even for things like LLVM it makes sense to keep tests unpinned: it makes them do their job of spotting regressions when the default is bumped.

Reply:
> I can't currently build this test, because of some broken dependency

It built when I prepared the patches initially. Right now, we can't test it because of some other dependencies apparently, so I'll leave it that way.
cc pleroma maintainers @picnoir @kloenk @yayayayaka 

> but I wonder whether we can change this to just postgresql instead, so we don't need to bump this every year?

To me it seemed that the easiest thing I could do was to go one version forward and see if the test builds.
If so, good (the rest is IMHO a problem for the maintainer). If not, the maintainers get a CC and that's it. But yeah, I guess I'll try the default package first next time.